assets/cubyz/biomes/cave/void.zig.zon

```diff
@@ -0,0 +1,14 @@
+.{
+	.isCave = true,
+	.maxHeight = -45000,
+	.minHeight = -50000,
+	.stoneBlock = "cubyz:air",
+	.fogDensity = 10,
+	.fogColor = 0x090511,
```

Comment:
I think the fog color should be bit brighter. With this you cannot really see a whole lot.

Reply:
We can make it a bit brighter.

Reply:
no more purple?

Reply:
I did kind of like the idea of making it purple. Have you tried a brighter purple? Mabye 0x553c64?
![Screenshot at 2024-12-19 17-20-32](https://github.com/user-attachments/assets/232226a2-5ed1-4188-b9c6-ce976a02d6a7)

Reply:
Hmm, I'm not sure, maybe if you made it a little darker?

Reply:
Yeah, I think that's going in the right direction. Maybe a little more saturation?
Also could you share a screenshot with a torch as a reference, (our eyes are too good at filtering out the color)

Reply:
I think the second one is good. @careeoki what do you think?

Reply:
Second looks good.